README.md

```
@@ -20,32 +20,62 @@
-| 일정 조율 생성                             | 일정 조율 생성 완료                            | 일정 조율 결과                                   |
-|-------------------------------------------|-------------------------------------------|-------------------------------------------|
-| <img src="https://github.com/user-attachments/assets/bd59de4a-afbc-4f17-9e02-51838348d9fd" width="1000"/> | <img src="https://github.com/user-attachments/assets/5c24bd2e-e212-407c-ac77-76ffc8c8c795" width="1000"/> | <img src="https://github.com/user-attachments/assets/ac61c6f0-3252-48dd-bcd9-838a6f9f034e" width="1000"/> |
-| 조율할 일정을 생성할 수 있습니다. | 일정 조율 링크를 복사하고 공유할 수 있습니다. | 필터링을 통해 원하는 사람들끼리의 일정 조율 결과를 확인할 수 있습니다. |
+### 개인 일정 관리
+#### 🔗 Google 캘린더 연동으로 더욱 편리하게
+사용자는 Google 캘린더와 연동하여 자신의 일정을 불러올 수 있어요.
+별도의 입력 없이 기존 일정을 기반으로 조율이 가능해요.
 
 | 내 일정 관리                    | 홈                    |
 |-------------------------------------------|-------------------------------------------|
 | <img src="https://github.com/user-attachments/assets/11449c72-9ee3-4cf1-8c2f-fc836109dcc6" width="1000"/> | <img src="https://github.com/user-attachments/assets/b2b64db2-802f-4b94-8026-2249b87ab7a4" width="1000"/> |  |
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)